shared/protocol/src/test/java/io/pravega/shared/protocol/netty/WireCommandsTest.java

```diff
@@ -35,7 +35,7 @@
 
```

Comment:
You've added three new wire commands. They should also have tests added to WireCommandsTest to make sure they round trip as expected.

Reply:
Fixed.